db/migrate/20180607233112_create_reviews.rb

```diff
@@ -1,5 +1,5 @@
 class CreateReviews < ActiveRecord::Migration[5.1]
-  def change
```

Comment:
I haven't tested these migrations myself but I assume that this would work. Not sure if having `drop_table` is the best idea to have in the migration itself though since people might not realise their table is being dropped.

Reply:
I can change this back. I've just been manually deleting the test.sqlite3 file and that actually works more thoroughly

Reply:
I think it'll be better to change this back, drop your db and start fresh. That should avoid having this problem for everyone from now on.